src/sections/pages/userDetail/index.js

```diff
@@ -0,0 +1,192 @@
+import OrdersTable from './ordersTable';
+import './styles.css';
+
+const orders = [
+    {
+        user: { name: 'Sara', photoUrl: '/assets/images/user2.jpg' },
+        status: 'VERIFIED',
+        amount: '$144.55',
+        date: '11/07/2018',
+    },
+    {
+        user: { name: 'Reinier', photoUrl: '/assets/images/user1.jpg' },
+        status: 'PROCESSING',
+        amount: '$285.05',
+        date: '11/07/2018',
+    },
+    {
+        user: { name: 'Leo', photoUrl: '/assets/images/user3.jpg' },
+        status: 'PROCESSING',
+        amount: '$125.25',
+        date: '11/06/2018',
+    },
+    {
+        user: { name: 'Tahimi', photoUrl: '/assets/images/user4.jpg' },
+        status: 'EXPIRED',
+        amount: '$104.55',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Pepe' },
+        status: 'VERIFIED',
+        amount: '$124.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Jose' },
+        status: 'VERIFIED',
+        amount: '$144.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Pedro' },
+        status: 'PROCESSING',
+        amount: '$94.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Juan' },
+        status: 'PROCESSING',
+        amount: '$124.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Pablo' },
+        status: 'EXPIRED',
+        amount: '$104.55',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Maria' },
+        status: 'VERIFIED',
+        amount: '$124.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Juana' },
+        status: 'VERIFIED',
+        amount: '$144.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Julio' },
+        status: 'PROCESSING',
+        amount: '$94.05',
+        date: '11/04/2018',
+    },
+    {
+        user: { name: 'Olga' },
+        status: 'PROCESSING',
+        amount: '$124.05',
+        date: '11/04/2018',
+    },
+];
+
+const userCardTitle = (
+    <span>
+        <FontAwesomeIcon icon={faFacebook} style={{ marginRight: '4px' }} />
+        Facebook user
+    </span>
+);
+
+const userCardAction = (
+    <ButtonGroup>
+        <ButtonIcon variant="border-filled" icon={<FontAwesomeIcon icon={faPlus} />} />
+        <ButtonIcon variant="border-filled" icon={<FontAwesomeIcon icon={faEdit} />} />
+    </ButtonGroup>
+);
+
+const userCardFooter = (
+    <div className="rainbow-align-content_space-between">
+        <div className="rainbow-flex">
+            <UserField label="Payments" value="Enabled" />
+            <div className="react-rainbow-admin-user-detail_vertical-divider" />
+            <UserField label="Total Balance" value="$ 100 000" />
+        </div>
+        <ButtonIcon icon={<FontAwesomeIcon icon={faAngleDown} />} />
+    </div>
+);
+
+export default class UserDetail extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
```

Comment:
when use componentDidMount to fetch data you can remove the constructor since  is useless